app/dashboard/templates/bounty/new_bounty.html

```diff
@@ -427,7 +431,7 @@ <h1 class="text-center">Fund Issue</h1>
                   <label class="font-caption letter-spacing text-black-60 text-uppercase" for="funderAddress">Funder Address</label>
                   <p class="font-body" v-show="funderAddressFallback === true">Please enter your Binance Wallet address. We are unable to validate a Binance address. Please confirm that you are using the correct Binance address.</p>
                   <input name="funderAddress" id="funderAddress" class="form__input" type="text" placeholder="Address with which the bounty will be paid out" v-model="form.funderAddress">
-                  <div class="text-danger" v-if="errors.funderAddress && (!form.funderAddress || chainId === '0')">
+                  <div class="text-danger" v-if="errors.funderAddress">
```

Comment:
why are we changing this ?

Reply:
funder address validation error is also needed for other chains (nervos/tezos) not only btc (0)